contracts/Views.cdc

```diff
@@ -0,0 +1,28 @@
+pub contract Views {
```

Comment:
Like my other comment, can we include a bunch of comments in the code here to show what each part is for?